docs/private-projection.md

```diff
@@ -0,0 +1,18 @@
+# Private Projection Support
+
+## Overview
+
+Support for private projections is an option on the C#/WinRT tool that allows for projections to be generated internal to the library or app. 
+This means you can keep uses of Windows SDK APIs internal to your project, and not expose these types to consumers of your library or app. 

```

Comment:
This is more you can keep uses of your component's projections internal to your project rather than the Windows SDK ones.